src/sdk/pynni/nni/constants.py

```diff
@@ -37,9 +37,9 @@
     'Evolution': 'EvolutionTuner',
     'SMAC': 'SMACTuner',
     'BatchTuner': 'BatchTuner',
+    'Medianstop': 'MedianstopAssessor',
```

Comment:
Better to put the Medianstop at the end separated by a blank line as original,  since it's an assessor and others're tuner

Reply:
Thank you for your kind suggestion.